tests/chart_tests/helm_template_generator.py

```diff
@@ -22,6 +22,7 @@
 from tempfile import NamedTemporaryFile
 from typing import Any, Dict, Tuple
 
+import jmespath
```

Comment:
FWIW I used to be so into using jmespath, but ran into some serious behavior weirdness that made me give up on it entirely. I describe the weirdness here: https://danielhoherd.com/tech-notes/jmespath/#quotes-matter